CMakeLists.txt

```diff
@@ -79,6 +79,11 @@ else ()
     set (MMK_ABI "arm")
     set (MMK_BITS 32)
     set (MMK_ARCH_ARM 1)
+  elseif (_ARCH MATCHES "armv7l")
```

Comment:
@rydb why not adding `v7l` to the `ARM32` regex above?

Reply:
@rydb friendly ping

Reply:
I may have pinged and not seen your first question.

Point is, `v7l` *is* a 32 bits ARM processor. So instead of replicating the if clause, consider adding `v7l` to the regex in line 26.